STEERING-COMMITTEE.md

```diff
@@ -46,6 +48,37 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
```

Comment:
This feels like the things which can be called "Knative" might be a moving target so I would suggest "define the criteria for which things can be called Knative"

Reply:
Rewrote per Alex's streamlined wording.